Docker-compose removed from GitHub runners

See https://github.com/actions/runner-images/issues/9692

this is causing failures:
```
 /home/runner/work/_temp/60b8c82b-4853-4552-a0ef-0f28ab3353ae.sh: line 1: docker-compose: command not found
Error: Process completed with exit code 127.
```

--- a/.github/workflows/tests.yaml
+++ b/.github/workflows/tests.yaml
@@ -133,10 +133,10 @@ jobs:
           GITHUB_USER: ${{ github.actor }}
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           CONTAINER: ${{ matrix.dockerfile }}
-        run: docker-compose build test-${{ env.CONTAINER }}
+        run: docker compose build test-${{ env.CONTAINER }}
 
       - name: Test ${{ matrix.name }}
-        run: docker-compose run test-${{ env.CONTAINER }}
+        run: docker compose run test-${{ env.CONTAINER }}
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           CONTAINER: ${{ matrix.dockerfile }}

